### DISCUSSION TITLE
manage remote podman using windows podman desktop tool

### DISCUSSION THREAD

**Discussion by MayilvahananT** [podman-desktop/podman-desktop#4477] [Ideas]
Hello team

we have 300 windows VDI VMs hosted in nutanix AHV. The VDI is using for developing code for all our developers. We are facing an issue to enable podman in vm, because it needs virtualization. Enabling virtualization over virtualization called nested visualization, if we enable nested virtual in all 200 VMs, we will loose vmotion capabilities. Hence, we have installed podman cli in VDI and connecting Linux based podman using ssh protocol. Now, developers are expecting the same capabilities from podman desktop also. In nutshell, podman desktop tool should have capabilities to manage remote podman. Please advise.

**Comment by rhatdan** [comment]
Why do you think podman needs virtualization?  It does not use kvm, by default?

**Comment by MayilvahananT** [comment]
Hi

If developer use windows 10/11 desktop operating system, then
virtualization is required (ie. Windows subsystem for Linux). The wsl is
equivalent to virtualization in windows. It it easy to introduce in
physical machines. But, we will loose vmotion feature if we enable in
windows 10/11 vm. If developers use Linux based operating system, this
challenge won’t be there. I think, you understood my concern.

**Comment by MayilvahananT** [comment]
The below screenshot is from Windows 10 Machine - Physical machine...We
have to enable wsl + vpm (windows subsystem for linux + virtual platform
manager)

[image: image.png]

For linux vm, machine is not required. it will work by default. Below is
screenshot for reference.

[image: image.png]

one more thing, I am able to communicate with linux containers using
podman-remote through the command line. My expectation is that the windows
based podman desktop should communicate remote linux based podman instances.

**Comment by afbjorklund** [comment]
* https://github.com/containers/podman-desktop/issues/279
